frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/OverviewSettings.tsx

```diff
@@ -76,7 +76,13 @@ export const OverviewSettings = ({
           <Text>{jt`Check out the ${(
             <ExternalLink
               key="doc"
-              href={`${docsUrl}?utm_source=${plan}&utm_media=static-embed-settings-overview`}
+              href={`${docsUrl}?${new URLSearchParams({
```

Comment:
This is a change for
Interactive embedding upsell
✅ Overview doc
Look and Feel doc
Look and Feel font upsell